scss/_variables.scss

```
@@ -1075,6 +1075,7 @@ $form-floating-input-padding-b:         .625rem !default;
 $form-floating-label-height:            1.5em !default;
 $form-floating-label-opacity:           .65 !default;
 $form-floating-label-transform:         scale(.85) translateY(-.5rem) translateX(.15rem) !default;
+$form-floating-label-color:             $input-placeholder-color !default; // When the label is acting as a placeholder
```

Comment can be removed. The naming of the Sass var is sufficient to understand what it does IMO.

```suggestion
$form-floating-label-color:             $input-placeholder-color !default;
```